### DISCUSSION TITLE
PETSc crash maybe?

### DISCUSSION THREAD

**Discussion by makeclean** [idaholab/moose#17382] [Q&A General] [answered]
I have a problem, I#m happy to share - it runs on 1120 MPI tasks when using presplit mode. If I dont pre-split, its a segfault on launch (presumably out of memory). If I run with 20 MPI tasks and 56 threads, it runs until after the 2nd timestep which is great 

```
mpirun noticed that process rank 5 with PID 226281 on node cpu-p-333 exited on signal 11 (Segmentation fault).
Time Step 2, time = 0.000585938, dt = 0.000390625
Computing initial residual ...........................                                     [ 28.34 s]
Updating displaced mesh ..........                                                         [ 11.14 s]
 0 Nonlinear |R| = 5.922167e+02
Updating displaced mesh .........                                                          [ 10.98 s]
At line 80 of file xerbla.f (unit = 6, file = 'stdout')
Fortran runtime error: Missing initial left parenthesis in format

At line 80 of file xerbla.f (unit = 6, file = 'stdout')
Fortran runtime error: Missing initial left parenthesis in format

At line 80 of file xerbla.f (unit = 6, file = 'stdout')
Fortran runtime error: Missing initial left parenthesis in format

At line 80 of file xerbla.f (unit = 6, file = 'stdout')
Fortran runtime error: Missing initial left parenthesis in format

--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun noticed that process rank 5 with PID 226281 on node cpu-p-333 exited on signal 11 (Segmentation fault).
```
Anyone seen anything like it?

**Comment by makeclean** [comment]
When it does run i get some lovely stress fields

![Screenshot from 2021-03-17 07-23-09](https://user-images.githubusercontent.com/2439754/111747805-ae3b0c80-8887-11eb-9e9c-2df7ae9291a7.png)

**Comment by makeclean** [comment, marked as answer]
Ahh its a blas error

**Reply by dschwen** [reply]
Looks like a runtime error trying to print the actual error.
https://www.netlib.org/lapack/explore-3.1.1-html/xerbla.f.html

My guess is, something is going horribly wrong way earlier, leading to bogus data going into some blas call.

**Comment by fdkong** [comment]
Did you get the error only when using threads?  How did you install blas?

**Reply by makeclean** [reply]
I couldnt actually run the problem without threads tbh

**Reply by fdkong** [reply]
Hmmm, I guess you should use a distributed mesh.

**Reply by makeclean** [reply]
It was running with a pre-splti mesh, but crashes, using mpi tasks, but fine with threading
